### PR TITLE
Update post-ti-indicator.md

### PR DESCRIPTION
**Pull request by msbemba**
added Generatealert parameter

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7305

File: microsoft-365/security/defender-endpoint/post-ti-indicator.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ expirationTime|DateTimeOffset|The expiration time of the indicator. **Optional**
 severity|Enum|The severity of the indicator. possible values are: "Informational", "Low", "Medium" and "High". **Optional**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/post-ti-indicator.md]:
```suggestion
severity|Enum|The severity of the indicator. Possible values are: "Informational", "Low", "Medium", and "High". **Optional**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - additional typo corrections, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**